go.mod

```diff
@@ -78,14 +78,20 @@ require (
 replace github.com/emicklei/go-restful => github.com/emicklei/go-restful v2.16.0+incompatible
 
 require (
-	cloud.google.com/go v0.102.1 // indirect
+	github.com/sigstore/cosign v1.11.1
+	github.com/sigstore/sigstore v1.4.0
+)
```

Comment:
Any reason to add them here and not with all the direct dependencies above?

Reply:
fixed, thank you.